Formula/qfs.rb

```diff
@@ -0,0 +1,37 @@
+class Qfs < Formula
+  desc "High-performance, fault-tolerant, distributed file system and its tools"
+  homepage "https://github.com/quantcast/qfs"
+  url "https://github.com/quantcast/qfs.git"
+  version "1.1.4-1"
```

Comment:
This needs to be a tarball rather than a Git checkout e.g. https://github.com/quantcast/qfs/archive/1.1.4.tar.gz If we're waiting for another release we can either apply patches that have been applied upstream or just wait for a new release.